core/utils.js

```diff
@@ -518,13 +527,55 @@ Blockly.utils.tokenizeInterpolation = function(message) {
         i--;  // Parse this char again.
         state = 0;
       }
+    } else if (state == 3) {
+      if ((c == '_') || ('0' <= c && c <= '9') || ('a' <= c && c <= 'z') ||
+          ('A' <= c && c <= 'Z')) {
+        buffer.push(c);
+      } else {
+        var key = buffer.join('');
+        var keyUpper = key.toUpperCase();
+        var bklyKey = keyUpper.startsWith('BKY_') ? keyUpper.substring(4) : null;
```

Comment:
This will fail in all versions of IE.  Use goog.string.startsWith instead.

Also, BKY_?  Wouldn't MSG_ be more intuitive?